tests/lib/globeCoordinate/globeCoordinate.GlobeCoordinate.tests.js

```diff
@@ -107,7 +126,9 @@ define( [
 				{ latitude: 1.00028, longitude: 0, precision: 1 / 3600 },
 				{ latitude: 1.0005, longitude: 0, precision: 1 / 36000 },
 				{ latitude: 89.9, longitude: -0.00031, precision: 1 / 3600000 },
-				{ latitude: 5, longitude: -0.00292, precision: 1 / 36000 }
+				{ latitude: 5, longitude: -0.00292, precision: 1 / 36000 },
+				{ latitude: 5, longitude: 2, globe: 'http://www.wikidata.org/entity/Q2' },
+				{ latitude: 5, longitude: 2, globe: 'http://www.wikidata.org/entity/Q313' }
```

Comment:
The test error is caused by a bug in equals. I will fix it in an other patch. To make your patch pass please add `precision: 1` to these lines.